### DISCUSSION TITLE
Troubleshoot help -- "ERROR: Unsupported startup parameter: _pq_.compression"

### DISCUSSION THREAD

**Discussion by dmgrnhse** [dbeaver/dbeaver#22476] [Q&A]
Hi all,

Would any of you know how to troubleshoot the following error - ERROR: Unsupported startup parameter: _pq_.compression
I'm not the most familiar with databases as I am learning SQL. This error began as I updated Dbeaver to it's most recent update.

**Comment by mkgrgis** [comment]
@dmgrnhse OS, DBMS name, some context ...

**Reply by dmgrnhse** [reply]
On a MAC (Sonoma 14.2.1), when attempting to connect to Redshift, I receive the following error in dbeaver.  This only began when I updated to the most recent version of dbeaver. No settings were changed aside from updating dbeaver. 

![dbeaver_error](https://github.com/dbeaver/dbeaver/assets/154359491/6250c21a-2da3-4436-87b6-074e49287ad1)
